custom_components/bambu_lab/pybambu/models.py

```diff
@@ -285,13 +320,15 @@ def print_update(self, data):
         self.remaining_time = data.get("mc_remaining_time", self.remaining_time)
         if data.get("gcode_start_time") is not None:
             self.start_time = start_time(int(data.get("gcode_start_time")))
+        # TODO: Check this logic actually works
+        if self.gcode_state is not 'Running':
```

Comment:
We might have some case inconsistency issues lurking here for this variable. I see for the localization changes this is expected to be lower case (maybe, not sure it does a case sensitive match) bit the actual mqtt content is all upper case. And we don't lower case it when we parse/save the string.